bcda/cclf/parser.go

```diff
@@ -26,6 +26,122 @@ func getCMSID(name string) (string, error) {
 	return parts[1], nil
 }
 
+func CheckIfAttributionCSVFile(filePath string) bool {
+	pattern := `P\.PCPB\.M\d{4}\.D\d{6}\.T\d{7}`
+	filenameRegexp := regexp.MustCompile(pattern)
+	found := filenameRegexp.Match([]byte(filePath))
+	return found
+}
+
+type CSVParser struct {
+	FilePath string
+}
+
+// func getACOConfigs() ([]service.ACOConfig, error) {
+// 	configs, err := service.LoadConfig()
+// 	if err != nil {
+// 		return []service.ACOConfig{}, err
+// 	}
+// 	return configs.ACOConfigs, err
+
+// }
+
+// type AttributionFileRegex struct {
+// 	fileNameReg string
+// 	regMatches  int
+// }
+
+// GetCSVMetadata builds an entry for the cclf_files table, based on the name of the file.
+// The filename regex is part of aco configuration.
+func GetCSVMetadata(path string) (csvFileMetadata, error) {
+	var metadata csvFileMetadata
+	var err error
+
+	// acos, err := getACOConfigs()
```

Comment:
I would like y'alls opinion on this commented out code. With these new attribution files, we currently don't have a way to identify who it's for; CCLF files in the function below contain the name of the ACO but this one will not. I would like to update our config file for ACOs and add a regular expression so that we can easily correlate the file here with TCOC, _without_ having to hardcode a solution. 

With the [current implementation](https://github.com/CMSgov/bcda-app/pull/1012/files#diff-2f11565dc82c9a1ee1b4f8daf8634313fc108c6e3e89c85d5806685259cea8d3R88), we can either build a struct of some sort with a regex pattern + ACO, or we can stick the regex in the config, which feels cleaner to me.